### ISSUE TITLE
Build Error on Arch Linux

### ISSUE THREAD

**Issue by ber532k** [swilmet/gCSVedit#4]
I was trying to get gCSVedit to build on Arch Linux, but I can't seem to figure out how to do it. Your README is apparently quite out of date, since `./configure` does not exist. If you could give me a hint as to how to get this building I'd be happy to create a pull request with an updated README after figuring stuff out myself.

**Comment by ber532k** [comment]
I just saw the note about running `autogen.sh`, so sorry about the above posting. Still I cant's seem to build either the latest release nor the current version on git. Running `./autogen.sh` produces the following error:

```
configure.ac:66: error: macro APPSTREAM_XML is not defined; is a m4 file missing?
/usr/share/aclocal/ax_require_defined.m4:35: AX_REQUIRE_DEFINED is expanded from...
configure.ac:66: the top level
autom4te: /usr/bin/m4 failed with exit status: 1
aclocal: error: echo failed with exit status: 1
```

Any help would be appreciated.

**Comment by swilmet-ucl** [comment]
Building from git is slightly more complicated. It's easier to build from the latest *.tar.xz tarball.

The build system of gCSVedit is similar to GNOME projects (that still use the Autotools).

But the best way to install gCSVedit is with Flatpak, I've created some scripts in the flatpak/ directory. I need to update the flatpak manifest (the *.json file). gCSVedit has been moved from sourceforge to github recently.

And gCSVedit git master doesn't build against Tepl git master (I'm also the Tepl developer), I need to update the code.

gCSVedit was just a small side project for me, don't expect too much from this app. It is not maintain as well as a GNOME app for example.
